projects/noloan/app/src/main/res/layout/lawsuit_fragment_form.xml

```diff
@@ -271,8 +287,8 @@
                             android:id="@+id/EditText_receivedSpamDate"
                             android:layout_width="match_parent"
                             android:layout_height="wrap_content"
-                            android:gravity="right"
                             android:focusable="false"
+                            android:gravity="right"
```

Comment:
What is this change?

Reply:
No change... I guess just the "line ordering"...